malariagen_data/anoph/sample_metadata.py

```diff
@@ -7,7 +7,7 @@
 from numpydoc_decorator import doc
 
 from ..util import check_types
-from . import base_params, map_params
+from . import base_params, het_params, map_params
```

Comment:
I don't think we should be importing `het_params` here. The `het_params` should go with the (yet to be created) `het_analysis` module which should be downstream of sample metadata. If we need the `single_sample` param here then I'd suggest moving that parameter up to `base_params`.

Reply:
Ha, OK. I think that's what I did the first time, and then changed my mind for some reason! I wasn't sure what `het_params` was for. Thanks. I'll see if I can.